R/oauth-token.R

```diff
@@ -32,7 +32,7 @@ oauth_token <- function(
 
   if (!is.null(expires_in)) {
     # Store as unix time to avoid worrying about type coercions in cache
-    expires_at <- as.numeric(.date) + expires_in
+    expires_at <- as.numeric(.date) + as.numeric(expires_in)
```

Comment:
Hmmm, I think the fix should probably be further upstream. And maybe `oauth_token()` should assert that `expires_in` is numeric?